### DISCUSSION TITLE
[Feature] Remote and Local server address for mobile apps

### DISCUSSION THREAD

**Discussion by nnmalex** [immich-app/immich#11942] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

With self-hosting Immich behind a reverse proxy it might be less performant (or have other restrictions, e.g. max file size) to always go via proxy address.
It would be great if we could (optionally) set up remote (public) server address alongside local address for the mobile app to determine if the latter is available at the moment and if it can be used instead of the former.
This is, for example, how Home Assistant apps do that.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#308

**Reply by nnmalex** [reply]
308 is closed as completed, but as far as I can see the feature is not available in the latest version of Immich. If this is rejected by the team can it be marked as such?
